zeitschrift-fur-digitale-geisteswissenschaften.csl

```diff
@@ -0,0 +1,389 @@
+        </names>
+      </if>
+      <else>
+        <choose>
+          <if type="chapter">
+            <names variable="editor">
+              <label prefix=" " form="verb-short" suffix=" "/>
+              <name delimiter=" / " delimiter-precedes-last="always"/>
+            </names>
+          </if>
+        </choose>
+      </else>
+    </choose>
+  </macro>
+  <macro name="secondary-container-contributors">
+    <choose>
+      <if variable="container-author" type="chapter" match="all">
+        <names variable="editor">
+          <label form="verb-short" text-case="lowercase" suffix=". "/>
+          <name delimiter=", " delimiter-precedes-last="always"/>
+        </names>
+      </if>
+      <else-if type="article-journal article-magazine" match="any">
+        <names variable="editor">
+          <label form="verb-short" text-case="lowercase" suffix=". "/>
+          <name delimiter=" /" delimiter-precedes-last="always"/>
+        </names>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="secondary-collection-contributors">
+    <names variable="collection-editor">
+      <name delimiter=" / " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="translatedby">
+    <names variable="translator">
+      <label form="verb-short" text-case="lowercase" suffix=". "/>
+      <name delimiter=", " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <label form="verb-short" prefix=". " suffix=" "/>
+      <name delimiter=" / " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="title-long">
+    <group>
+      <choose>
+        <if type="book">
+          <text variable="title" form="long"/>
+          <text macro="editor"/>
+        </if>
+      </choose>
+      <choose>
+        <if type="webpage">
+          <text variable="title" form="long"/>
+        </if>
+      </choose>
+      <choose>
+        <if type="personal_communication">
+          <text variable="title" form="long"/>
+        </if>
+      </choose>
+      <choose>
+        <if type="post-weblog">
+          <text variable="title" form="long"/>
+        </if>
+      </choose>
+      <choose>
+        <if type="chapter">
+          <text variable="title" form="long"/>
+        </if>
+      </choose>
+      <choose>
+        <if type="article-journal">
+          <text variable="title" form="long" suffix="."/>
+        </if>
+      </choose>
+      <choose>
+        <if type="book">
+          <group>
+            <choose>
+              <if variable="volume">
+                <text prefix=". " term="volume" form="short" suffix=".&#160;"/>
+                <text variable="volume"/>
+              </if>
+            </choose>
+          </group>
+        </if>
+      </choose>
+      <text prefix=" / " macro="secondary-contributors"/>
+    </group>
+  </macro>
+  <macro name="title-short">
+    <group delimiter=". ">
```

Comment:
weder die Gruppe noch das `choose/if` is hier notwendig.

Reply:
Das habe ich nun auskommentiert, da title-short keine weitere Anwendung im Stil hat.

Reply:
title-short habe ich rausgenommen, da es keine Verwendung im Stil findet.